app/models/member.rb

```diff
@@ -86,4 +88,10 @@ def token_payload
       authentication_id: authentication.try(:id)
     }
   end
+
+  private
+
+  def existing_memberships_and_email_changed?
+    Member.where(email: email).where.not(id: id).any? && email_changed?
```

Comment:
@osahyoun I'm not sure I understand what you're doing here. I think you're actually validating uniqueness only if the email is already unique, which is redundant and makes the validation have no effect.

I believe we should leave the validation in place without any `if`s, but before deploying make sure we remove any duplicates from the database.